staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3773,6 +3780,7 @@ type ReplicationControllerSpec struct {
 	// controller, if empty defaulted to labels on Pod template.
 	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
 	// +optional
+	// +mapType=atomic
```

Comment:
This is covered by https://github.com/kubernetes/kubernetes/pull/97989/

Reply:
Yes, I'll drop everything already in #97989 from this PR. Probably makes sense for #97989 to cover all the map typed fields.